amd_openvx_extensions/amd_rpp/source/tensor/CropMirrorNormalize.cpp

```diff
@@ -27,9 +27,9 @@ struct CropMirrorNormalizeLocalData {
     Rpp32u deviceType;
     RppPtr_t pSrc;
     RppPtr_t pDst;
-    vx_float32 *pMultiplier;
-    vx_float32 *pOffset;
-    vx_uint32 *pMirror;
+    Rpp32f *pMultiplier;
```

Comment:
I think it is better to use vx_types for OpenVX

Reply:
Changed to vx_types